app/components/category-item.js

```diff
@@ -24,7 +24,7 @@ const {
  */
 export default Component.extend({
   classNames: ['category-item'],
-  classNameBindings: ['selected'],
+  classNameBindings: ['selected', 'category.slug'],
```

Comment:
I'm going to rework this some myself, but with BEM we would rather have it be something like `--selected` with an existing class rather than adding the `.selected` class onto this. The namespacing is really critical here to ensuring that our styles don't overlap. I know you're just using the existing class, but this should probably be in the refactor.